sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -485,6 +500,31 @@ def text(self, encoding=None):
             encoding = "utf-8-sig"
         return self.body().decode(encoding)
 
+    def _decode_parts(self, message, http_response_type, requests):
+        # type: (Message, Type[_HttpResponseBase], List[HttpRequest]) -> List[HttpResponse]
+        """Rebuild an HTTP response from pure string."""
+        responses = []
+        for index, raw_reponse in enumerate(message.get_payload()):
+            content_type = raw_reponse.get_content_type()
+            if content_type == "application/http":
+                responses.append(
+                    _deserialize_response(
+                        raw_reponse.get_payload(decode=True),
+                        requests[index],
+                        http_response_type=http_response_type,
+                    )
+                )
+            elif content_type == "multipart/mixed" and requests[index].multipart_mixed_info:
+                # The message batch contains one or more change sets
+                changeset_requests = requests[index].multipart_mixed_info[0]  # type: ignore
+                changeset_responses = self._decode_parts(raw_reponse, http_response_type, changeset_requests)
+                responses.extend(changeset_responses)
+            else:
+                raise ValueError(
+                    "Multipart doesn't support part other than application/http for now"
```

Comment:
Guess the message should add multipart/mixed ?

Reply:
hmm I wondered about this - however it's technically still true that the individual multi-parts themselves must still be application/http. And I think if you got this error, and it mentioned multipart/mixed - that might be confusing.